### DISCUSSION TITLE
Error 401 when trying to edit file in MS learn documentation repo

### DISCUSSION THREAD

**Discussion by HerbertMauerer** [community/community#60315] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello,

I used to be able to directly edit and submit PR for Microsoft public docs like:
https://learn.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/accounts-block-microsoft-accounts

When I click edit, I get this error since last week:
Error loading page
An unexpected 401 error occured. Try reloading the page.

A colleague was able to submit a change for this file. is this really access denied?

Cheers,

Herbert

**Comment by HerbertMauerer** [comment]
it works now. not sure someone worked on the solution...

**Comment by NuAlex** [comment]
I'm having the same issue now - says:
Error loading page
An unexpected 401 error occured. Try reloading the page.

- Does it still work for you @HerbertMauerer ?

**Reply by HerbertMauerer** [reply]
Hello,

it is back now. Really annoying when you used to be able to create DOC changes directly...
